Zenith/Controllers/EventsController.cs

```diff
@@ -22,7 +24,57 @@ public class EventsController : Controller
             var events = db.Events.Include(E => E.ActivityCategory);
             return View(await events.ToListAsync());
         }
+        public async Task<ActionResult> Schedule()
+        {
+            return View(getDic());
+        }
+        public ScheduleViewModel getDic()
```

Comment:
Minor but method name should start with a capital letter to be consistent looool